src/models/List.ts

```diff
@@ -57,15 +57,15 @@ export default class List<T> extends Array {
     const list = new List();
     list.links = _links;
     list.count = count;
-    list.nextPage = async () =>
+    list.nextPage = async (): Promise<any> =>
```

Comment:
i think it's void?  `Promise<void>`

Reply:
It's actually:

```
    list.nextPage = async (): Promise<List<Model>> =>
```

I've updated this in the code.